Hi @Iamtheboneofmysword,
that's great, thanks!
It's nice to hear you're using visualpic with osiris. It's been a while since we've used osiris here, so I wasn't very confident that the reader was still working.

Could you fix the small formatting problems that flake8 is raising?
```
./visualpic/ui/basic_render_window.py:98:80: E501 line too long (80 > 79 characters)
./visualpic/ui/basic_render_window.py:99:80: E501 line too long (80 > 79 characters)
./visualpic/ui/controls/mpl_figure_with_draggable_points.py:36:22: E128 continuation line under-indented for visual indent
./visualpic/ui/controls/mpl_figure_with_draggable_points.py:37:22: E128 continuation line under-indented for visual indent
./visualpic/ui/controls/mpl_figure_with_draggable_points.py:38:22: E128 continuation line under-indented for visual indent
```
After that I'll merge the PR.

--- a/visualpic/ui/basic_render_window.py
+++ b/visualpic/ui/basic_render_window.py
@@ -8,7 +8,6 @@ Copyright 2016-2020, Angel Ferran Pousa.
 License: GNU GPL-3.0.
 """
 
-
 from pkg_resources import resource_filename
 import platform
 import ctypes
@@ -24,8 +23,9 @@ from visualpic.ui.controls.qt.QVTKRenderWindowInteractor import (
 from visualpic.ui.setup_field_volume_window import SetupFieldVolumeWindow
 from visualpic.ui.setup_scatter_species_dialog import SetupScatterSpeciesDialog
 from visualpic.ui.render_settings_dialog import RenderSettingsDialog
-from visualpic.helper_functions import (
-    get_closest_timestep, get_previous_timestep, get_next_timestep)
+from visualpic.helper_functions import (get_closest_timestep,
+                                        get_previous_timestep,
+                                        get_next_timestep)
 
 # code for proper scaling in high-DPI screens. Move this somewhere else once \
 # final UI is implemented.
@@ -40,7 +40,6 @@ QtWidgets.QApplication.setStyle(QStyleFactory.create('Fusion'))
 
 
 class BasicRenderWindow(QtWidgets.QMainWindow):
-
     """Basic Qt window for interactive visualization of 3D renders."""
 
     def __init__(self, vtk_visualizer, parent=None, window_size=None):
@@ -95,8 +94,9 @@ class BasicRenderWindow(QtWidgets.QMainWindow):
         self.hl.addWidget(self.next_button)
         self.timestep_slider = QtWidgets.QSlider(Qt.Horizontal)
         if len(self.available_timesteps) > 0:
-            self.timestep_slider.setRange(int(np.min(self.available_timesteps)),
-                                          int(np.max(self.available_timesteps)))
+            self.timestep_slider.setRange(
+                int(np.min(self.available_timesteps)),
+                int(np.max(self.available_timesteps)))
             self.timestep_slider.setValue(int(self.vtk_vis.current_time_step))
         else:
             self.timestep_slider.setEnabled(False)

--- a/visualpic/ui/controls/mpl_figure_with_draggable_points.py
+++ b/visualpic/ui/controls/mpl_figure_with_draggable_points.py
@@ -32,10 +32,11 @@ class FigureWithDraggablePoints(Figure):
         self.patch_color = patch_color
         self.drag_points = {}
         self.histograms = {}
-        super().__init__(figsize=figsize, dpi=dpi, facecolor=facecolor,
-                     edgecolor=edgecolor, linewidth=linewidth,
-                     frameon=frameon, subplotpars=subplotpars,
-                     tight_layout=tight_layout)
+        super().__init__(
+            figsize=figsize, dpi=dpi, facecolor=facecolor,
+            edgecolor=edgecolor, linewidth=linewidth,
+            frameon=frameon, subplotpars=subplotpars,
+            tight_layout=tight_layout)
         self.create_axes(nrows, ncols, xlabels, ylabels, share_x_axis,
                          share_y_axis, hist, hist_edges)
 

